samza-kv-inmemory/src/main/java/org/apache/samza/storage/kv/inmemory/InMemoryTableDescriptor.java

```diff
@@ -40,6 +41,13 @@ public InMemoryTableDescriptor(String tableId) {
     super(tableId);
   }
 
+  /**
+   * {@inheritDoc}
+   */
+  public InMemoryTableDescriptor(String tableId, KVSerde<K, V> serde) {
```

Comment:
I think inheritDoc doesn't work for constructors (since they're not overrides). Same comment for other classes as well.